catalogue_pipeline/id_minter/src/main/scala/uk/ac/wellcome/platform/idminter/database/IdentifiersDao.scala

```diff
@@ -71,11 +73,13 @@ class IdentifiersDao @Inject()(db: DB, identifiers: IdentifiersTable)
             )
         }.update().apply()
       }
+    } recover {
+      case e: SQLIntegrityConstraintViolationException =>
+        warn(s"Failed inserting identifier $identifier", e)
```

Comment:
Could we get a more descriptive log message please? Maybe something like:

```scala
warn(s"Unable to insert $identifier because of integrity constraints", e)
```

to distinguish it from the message below.